opo-makefile.mk

```diff
@@ -65,11 +65,11 @@ endif
 
 .PHONY: opo-test-e2e-up-local
 opo-test-e2e-up-local: opo-test-clean
-	operator-sdk test local ./test/e2e/ --up-local --namespace openshift-pipelines  --go-test-flags "-v -timeout=10m" --local-operator-flags "--recursive"
+	operator-sdk test local ./test/e2e/ --up-local --operator-namespace openshift-pipelines  --go-test-flags "-v -timeout=10m" --local-operator-flags "--recursive"
```

Comment:
could you add `--watch-namespace ""` as well
we need both.

from v17.2 we have 2 flags instead of the old `--namespace` as you noticed.

if we don't set `--watch-namespace=""` the operator under test will only watch one namesapce, that will be = value of `--operator-namespace`

Reply:
Updated Thank you